src/Table/index.tsx

```diff
@@ -8,42 +8,92 @@ export interface TableProps {
   }
   rows: RowValue[][]
   headers: string[]
+  isPage?: boolean
 }
 
 type RowValue = React.ReactNode | number | string | null | undefined
-const defaultStyle = `color: var(--Black, #000);font-family: Roboto Mono;font-size: 16px;font-style: normal;text-align: left;font-weight: 400;line-height: normal;`
-
-const Table: React.FC<React.PropsWithChildren<TableProps>> = (props) => (
-  <Wrapper {...props}>
-    <TR key={Math.random().toString()} style={props.styles?.tr}>
-      {props.headers.map((header: string) => (
-        <TH key={Math.random().toString()} style={props.styles?.th || {}}>
-          {header}
-        </TH>
-      ))}
-    </TR>
-    {props.rows.map((row: RowValue[]) => (
-      <TR key={Math.random().toString()} style={props.styles?.tr}>
-        {row.map((value: RowValue) => (
-          <TD key={Math.random().toString()}>{value}</TD>
+
+const Table: React.FC<React.PropsWithChildren<TableProps>> = (props) => {
+  return (
+    <Background>
+      <Wrapper {...props}>
+        <TR
+          color={'not-page'}
+          key={Math.random().toString()}
+          style={props.styles?.tr}
+        >
+          {props.headers.map((header: string) => (
+            <TH key={Math.random().toString()} style={props.styles?.th || {}}>
+              {header}
+            </TH>
+          ))}
+        </TR>
+        {props.rows.map((row: RowValue[]) => (
+          <TR key={Math.random().toString()} style={props.styles?.tr}>
+            {row.map((value: RowValue) => (
+              <TD key={Math.random().toString()}>{value}</TD>
+            ))}
+          </TR>
         ))}
-      </TR>
-    ))}
-  </Wrapper>
-)
+      </Wrapper>
+    </Background>
+  )
+}
 
 const TH = styled('th')`
-  ${defaultStyle}
-  text-decoration-line: underline;
+  border-bottom: solid 1px #fff;
+  text-align: left;
+  padding: 2px 5px;
+  border-left: solid 1px #fff;
+  background-color: #27847a;
+  color: #fff;
+
+  font-family: Roboto;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 26x; /* 171.429% */
 `
+
 const TD = styled('td')`
-  ${defaultStyle}
+  color: #1a1a1a;
+
+  font-family: Inter;
+  font-size: 26px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 0px; /* 0% */
+  padding: 5px;
+  height: 60px;
+  text-align: center;
+`
```

Comment:
I've just noticed these styles now replace `defaultStyles` and they are different. I much prefer this but do we know where this component is being used and if this will have any serious affect?

Reply:
Yea, fortunately this wasn't used anywhere and I've noticed that we have more time for this project so I've refactored/abstracted into `styles-map.tsx` where we can speficy all sorts of table styles e.g. TR/TH/TD